### PR TITLE
[AU4] Add track button styling

### PR DESCRIPTION
**Pull request by grliszas14**
Resolves: https://github.com/audacity/audacity/issues/7044
Resolves: https://github.com/audacity/audacity/issues/7043

### CODE REVIEW

File: au4/src/projectscene/qml/Audacity/ProjectScene/trackspanel/TracksTitleBar.qml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        mouseArea.acceptedButtons: Qt.NoButton
+
+        transparent: false
+        contentItem: StyledIconLabel {
+            iconCode: IconCode.TOOLBAR_GRIP
+        }
+    }
+
+    Rectangle {
+        id: spacer
```
Comment (Eism) [inline au4/src/projectscene/qml/Audacity/ProjectScene/trackspanel/TracksTitleBar.qml:44]: I would recommend using RowLayout + SeparatorLine

Comment (grliszas14) [reply au4/src/projectscene/qml/Audacity/ProjectScene/trackspanel/TracksTitleBar.qml]: fixed

File: au4/src/projectscene/qml/Audacity/ProjectScene/trackspanel/TracksTitleBar.qml [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+            transparent: false
+            contentItem: StyledIconLabel {
+                iconCode: IconCode.TOOLBAR_GRIP
+            }
+        }
+
+        SeparatorLine {
+            id: spacer
+
+            height: 32
```
Comment (Eism) [inline au4/src/projectscene/qml/Audacity/ProjectScene/trackspanel/TracksTitleBar.qml:53]: oh, you don't need these settings, they are already in the component
try using empty `SeparatorLine { }`

Comment (grliszas14) [reply au4/src/projectscene/qml/Audacity/ProjectScene/trackspanel/TracksTitleBar.qml]: fixed

**Review by Eism — APPROVED** [review]

**Pull request merged**